distribution/tools/plugin-cli/src/main/java/org/elasticsearch/plugins/cli/InstallPluginAction.java

```diff
@@ -359,7 +364,7 @@ boolean isSnapshot() {
      */
     private String getElasticUrl(
         final String stagingHash,
-        final Version version,
+        final String version,
```

Comment:
Nit: can we remove this parameter? I find it strange we pass it down and then in the body we sometimes use it, sometimes Build.current() (and it is always some Build.current() flavour)